src/compute/arithmetics/timestamp.rs

```diff
@@ -0,0 +1,351 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could we remove this header? This is by default Apache 2.0 as per README; Let's make it ASF if/when this gets donated.